examples/showcase/where_is_stereo.py

```diff
@@ -0,0 +1,154 @@
+periods = [100, 180, 120]  # may need to be updated as orbits evolve
+mission_coords = {mission: get_horizons_coord(mission, {'start': obstime,
+                                                        'stop': obstime + period * u.d,
+                                                        'step': '1d'})
+                  for mission, period in zip(missions, periods)}
+
+##############################################################################
+# Define a convenience function for converting coordinates to plot positions.
+
+frame = HeliocentricEarthEcliptic(obstime=obstime)
+
+
+def coord_to_heexy(coord):
+    coord = coord.transform_to(frame)
+    coord.representation_type = 'cartesian'
+    return coord.y.to_value('AU'), coord.x.to_value('AU')
+
+##############################################################################
+# Set Matplotlib settings to the desired appearance and initialize the axes.
+
+
+mpl.rcParams.update({'figure.facecolor': 'black',
+                     'axes.edgecolor': 'white',
+                     'axes.facecolor': 'black',
+                     'axes.labelcolor': 'white',
+                     'axes.titlecolor': 'white',
+                     'lines.linewidth': 1,
+                     'xtick.color': 'white',
+                     'xtick.direction': 'in',
+                     'xtick.top': True,
+                     'ytick.color': 'white',
+                     'ytick.direction': 'in',
+                     'ytick.right': True})
+
+fig = plt.figure()
+ax = fig.add_subplot()
+
+ax.set_xlim(-2.15, 2.15)
+ax.set_xlabel('Y (HEE)')
+ax.xaxis.set_major_locator(MultipleLocator(1))
+ax.xaxis.set_minor_locator(MultipleLocator(0.1))
+
+ax.set_ylim(1.8, -1.8)
+ax.set_ylabel('X (HEE)')
+ax.yaxis.set_major_locator(MultipleLocator(1))
+ax.yaxis.set_minor_locator(MultipleLocator(0.1))
+
+ax.set_title(obstime.strftime('%d-%b-%Y %H:%M UT'))
+ax.set_aspect('equal')
+
+# sphinx_gallery_defer_figures
+
+##############################################################################
+# Draw the Sun-Earth line.
+
+ax.plot([0, 0], [0, 2], linestyle='dotted', color='gray')
+
+# sphinx_gallery_defer_figures
+
+##############################################################################
+# Draw Mercury, Venus, Earth, and Mars, with Earth formatted differently.
+
+for planet, coord in planet_coords.items():
+    ax.plot(*coord_to_heexy(coord), linestyle='dashed', color='gray')
+
+    if planet == 'Earth':
+        color, markersize = 'lime', 10
+    else:
+        color, markersize = 'gray', None
+
+    x, y = coord_to_heexy(coord[0])
+    ax.plot(x, y, 'o', markersize=markersize, color=color)
+    ax.text(x + (0.1 if planet == 'Earth' else 0.05), y, planet, color=color)
+
+# sphinx_gallery_defer_figures
+
+##############################################################################
+# Draw the STEREO spacecraft (sans orbits), as well as Sun-STEREO lines.
```

Comment:
```suggestion
# Draw the STEREO spacecraft (without orbits), as well as Sun-STEREO lines.
```

Reply:
Weird, I don't normally ever use the word "sans", so I don't know why I did in this case...

Reply:
Done